pkg/health/health.go

```diff
@@ -0,0 +1,51 @@
+package health
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spiffe/spiffe-helper/cmd/spiffe-helper/config"
+	"github.com/spiffe/spiffe-helper/pkg/sidecar"
+)
+
+func StartHealthServer(configFile string, daemonModeFlag bool, log logrus.FieldLogger, sidecar *sidecar.Sidecar) error {
+	hclConfig, err := config.ParseConfigFile(log, configFile, daemonModeFlag)
+	if err != nil {
+		return err
+	}
+
+	if err := hclConfig.ValidateConfig(log); err != nil {
+		return fmt.Errorf("invalid configuration: %w", err)
+	}
```

Comment:
I think we should just parse the config once in main and pass the config to this pkg

Reply:
Good idea. Fixed.